client/homebrew/editor/snippetbar/snippets/watercolor.gen.js

```diff
@@ -0,0 +1,20 @@
+const _ = require('lodash');
+
+const watercolorBG = [
+	'https://i.imgur.com/UPUX4zG.png',
+	'https://i.imgur.com/Q6Uquv9.png',
+	'https://i.imgur.com/UOtaJpJ.png',
+	'https://i.imgur.com/Wy2DVk3.png',
+	'https://i.imgur.com/IempVlg.png',
+	'https://i.imgur.com/iJ1ddgd.png',
+	'https://i.imgur.com/qg4an04.png',
+	'https://i.imgur.com/ogGTcCh.png',
+	'https://i.imgur.com/itBD19A.png',
+	'https://i.imgur.com/tbbycAt.png',
+	'https://i.imgur.com/1SVWpGR.png',
+	'https://i.imgur.com/wC1zkuJ.png'
+];
+
+module.exports = ()=>{
+	return `\n![watercolor bg](${_.sample(watercolorBG)}) {watercolor,position:absolute,top:0px,left:0px,width:240px,filter:hue-rotate(0deg),mix-blend-mode:multiply,opacity:80%}\n`;
```

Comment:
Ooooh...... I really like the div background approach.  I tinkered around with it a bit and got this (as a Span instead of a Div):

```html
{{watercolor1,top:0px,left:0px,height:300px,background-color:#b0720b,opacity:100%}}
```

```css
[class*="watercolor"] {
	position:absolute;
	width:800px;
	height:800px;
	-webkit-mask-image:var(--wc);
	-webkit-mask-size:contain;
	-webkit-mask-repeat:no-repeat;
	mask-image:var(--wc);
	mask-size:contain;
	mask-repeat:no-repeat;
	background-color:red;
	--wc : url(https://i.imgur.com/ogGTcCh.png); /*default image*/
	z-index:-2;
}

.watercolor1 {
	--wc : url(https://i.imgur.com/ogGTcCh.png);
}

.watercolor2 {
	--wc : url(https://i.imgur.com/UPUX4zG.png);
}

.watercolor3 {
	--wc : url(https://i.imgur.com/Q6Uquv9.png);
}

.watercolor4 {
	--wc : url(https://i.imgur.com/UOtaJpJ.png);
}
```

So, we let them select the background image by changing the number in the classname (an invalid number will just display a "default" image). This way the don't have to "chance" upon the randomly-generated one from the snippet. We also give a default "color" which we could make a standard "coffee-stain" color.

Further. the user isn't exposed to any custom `--wc` variables; it's all just standard CSS options, and they can directly set a color from the full range of hue, saturation, value, instead of just the hue rotation.

True, they can't now inject their own images because it would not match the `mask`, but if a user wants custom watermarks they could just use a standard background image to do that.

I also removed these lines because I couldn't see that they did anything after making these changes.
```
	background-repeat:no-repeat;
	background-blend-mode:overlay;
```

There's even some fun stuff we could do with like radial gradients and stuff to kind of "burn" or "fade" the edges if we add something like this:

```
background-image: radial-gradient(circle, rgba(0,0,0,0) 40%, #700);
```